lib/dao.js

```diff
@@ -1897,6 +1941,7 @@ DataAccessObject.updateAll = function (where, data, options, cb) {
     try {
       where = removeUndefined(where);
       where = Model._coerce(where);
+      var currentInstance = data;
```

Comment:
`updateAll` is possibly modifying multiple instances, therefore `context.currentInstance` should not be set.